angular.json

```
@@ -81,8 +78,524 @@
+          "options": {
+            "tsConfig": [
+              "modules/router-store/tsconfig-build.json",
+              "modules/router-store/tsconfig.spec.json"
+            ],
+            "exclude": ["**/node_modules/**", "!modules/router-store/**"]
+          }
+        },
+        "test": {
+          "builder": "@nrwl/jest:jest",
+          "options": {
+            "jestConfig": "modules/router-store/jest.config.js",
+            "tsConfig": "modules/router-store/tsconfig.spec.json",
+            "setupFile": "modules/router-store/test-setup.ts"
+          }
+        }
+      },
+      "schematics": {}
+    },
+    "schematics": {
+      "root": "modules/schematics",
+      "sourceRoot": "modules/schematics/src",
+      "projectType": "library",
+      "schematics": {},
+      "architect": {
+        "lint": {
+          "builder": "@angular-devkit/build-angular:tslint",
+          "options": {
+            "tsConfig": [
+              "modules/schematics/tsconfig.lib.json",
+              "modules/schematics/tsconfig.spec.json"
+            ],
+            "exclude": ["**/node_modules/**", "!modules/schematics/**"]
+          }
+        },
+        "test": {
+          "builder": "@nrwl/jest:jest",
+          "options": {
+            "jestConfig": "modules/schematics/jest.config.js",
+            "tsConfig": "modules/schematics/tsconfig.spec.json",
+            "passWithNoTests": true
+          }
+        },
+        "build": {
+          "builder": "@nrwl/node:package",
+          "options": {
+            "outputPath": "dist/modules/schematics",
+            "tsConfig": "modules/schematics/tsconfig-build.json",
+            "packageJson": "modules/schematics/package.json",
+            "main": "modules/schematics/src/index.ts",
+            "sourceMap": false,
+            "assets": [
+              "collection.json",
+              {
+                "input": "./modules/schematics/src",
+                "glob": "**/*.!(ts)",
+                "output": "./src"
+              },
+              {
+                "input": "./modules/schematics",
+                "glob": "collection.json",
+                "output": "."
+              }
+            ]
+          }
+        },
+        "build-release": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "parallel": false,
+            "commands": [
+              {
+                "command": "ng build schematics"
+              },
+              {
+                "command": "yarn tsc -p modules/schematics/tsconfig.schematics.json"
+              },
+              {
+                "command": "cpy modules/schematics/collection.json dist/modules/schematics"
+              },
+              {
+                "command": "cpy modules/schematics/migrations/migration.json dist/modules/schematics/migrations"
+              },
+              {
+                "command": "cpy modules/schematics/src/**/files/**/* dist/ --parents"
```

One more Windows nit... it seems like we also have to provide a file extension here 
```suggestion
                "command": "cpy modules/schematics/src/**/files/**/*.* dist/ --parents"
```

---

Got it